lib/portlayer/exec/container.go

```diff
@@ -483,9 +483,11 @@ func (c *Container) stop(ctx context.Context, waitTime *int32) error {
 					log.Infof("power off %s task skipped due to guest shutdown", c.ExecConfig.ID)
 					return nil
 				}
-			}
+				log.Warnf("hard power off failed due to: %#v", terr)
```

Comment:
might use another message, to show the error type GenericVmConfigFault, similar to the InvalidPowerState.